### ISSUE TITLE
Bug with Metrics/ModuleLength

### ISSUE THREAD

**Issue by justin808** [bbatsov/rubocop#1948]
Problem with latest release 0.31.0.

This problem does not show on prior versions.

We're not customizing this.

```
Offenses:
app/models/importers/import_utility.rb:10:3: C: Metrics/ModuleLength: Module definition is too long. [166/100]
  module ImportUtility
  ^^^^^^
spec/adv_import/mappers/fixtures.rb:1:1: C: Metrics/ModuleLength: Module definition is too long. [602/100]
module AdvImport::Fixtures
^^^^^^
```

Here's the files

```ruby
module Importers
  module ImportUtility
```

and 
```ruby
module AdvImport::Fixtures
```

**Comment by jonas054** [comment]
I get the impression that you think the offenses are related to the names of the modules, but it's about the number of source code lines inside them.

**Comment by bquorning** [comment]
Reminds me of #1851. Maybe we could change the message to mention that it’s about the number of lines in the module/class?

**Comment by jonas054** [comment]
Yeah, the `MethodLength` cop says "Method has too many lines". Let's change `ClassLength` and `ModuleLength` to use the same wording.

**Comment by bquorning** [comment]
@jonas054 I’ll open a PR.

**Comment by bquorning** [comment]
Fixed in #1950, can be closed now.
